pkg/api/errors/transaction.go

```diff
@@ -0,0 +1,71 @@
+package errors
+
+import (
+	"github.com/wavesplatform/gowaves/pkg/proto"
+	"net/http"
+)
+
+type transactionError struct {
+	genericError
+}
+
+// TODO(nickeskov): IMPLEMENT ME
```

Comment:
should it be implemented if it's not used?

Reply:
Yes because we need to support scala node HTTP API.